config/manager/manager.yaml

```diff
@@ -1,60 +1,55 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: odh-model-controller
-  name: system
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: odh-model-controller
-  namespace: system
   labels:
     control-plane: odh-model-controller
+    app: odh-model-controller
 spec:
   selector:
     matchLabels:
       control-plane: odh-model-controller
-  replicas: 1
+  replicas: 3
```

Comment:
Replicas changed to 3. Is this intentional?

Reply:
Yes, odh-manifests uses 3 for a long time.